website_field_autocomplete/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Website Field - AutoComplete
+============================
+
+Adds an AutoComplete field for use in Odoo Website.
+
+This module is somewhat difficult to use on its own in an effort to not require
+any dependencies other than website.
+
+Usage
+=====
+
+To use this module, you need to add an input field with the
+``s_website_autocomplete`` class, such as in the below format:
+
+    <input type="text"
+           class="s_website_autocomplete"
+           data-queryField="name"
+           data-displayField="name"
+           data-limit="5"
+           data-domain='[["website_published", "=", true]]'
+           data-model="product.template"
+           />
+
+
++-------------------+--------------------------------------------+------------+----------+
+|  Attribute        |  Description                               |  Default   | Required |
++===================+============================================+============+==========+
+| data-model        | Model name to query                        |            | True     |
++-------------------+--------------------------------------------+------------+----------+
+| data-queryField   | Field to query when searching              | name       | False    |
++-------------------+--------------------------------------------+------------+----------+
+| data-displayField | Field to display (defaults to queryField)  | queryField | False    |
```

Comment:
I'd prefer `data-query-field` and `data-display-field`. Personal liking though, you are not required to satisfy it :yum: